### PR TITLE
docs: added debugging tips while installing SDK for the first time

### PR DESCRIPTION
**Pull request by petermin123**
# The problem
When I tried to install the SDK during onboarding, I encountered issues regarding Python versions and dependency conflicts. Thanks to help from Alex and Sebastian, I was able to eventually fix this issue, and I would like to add this debugging tip to the installation page.

Supporting Slack thread: https://zapatista.slack.com/archives/C01LHU1E6DB/p1718789163191309?thread_ts=1718739278.973009&cid=C01LHU1E6DB

# This PR's solution
This PR helps address this issue revolving virtual environment and specifies the recommended version for installing the Orquestra SDK.

Screenshot of the added block quote:
Screenshot 2024-07-03 at 4 38 21 PM

# Checklist

_Check that this PR satisfies the following items:_

- [x] Tests have been added for new features/changed behavior (if no new features have been added, check the box).
- [x] The [changelog file](CHANGELOG.md) has been updated with a user-readable description of the changes (if the change isn't visible to the user in any way, check the box).
- [x] The PR's title is prefixed with `<feat/fix/chore/imp[rovement]/int[ernal]/docs>[!]:`
- [x] The PR is linked to a JIRA ticket (if there's no suitable ticket, check the box).

### CODE REVIEW

File: projects/orquestra-sdk/docs/tutorials/installing-macos-linux.rst [hunk 1]
```diff
@@ -18,6 +18,17 @@ Install the Orquestra Workflow SDK by running:
 This will install ``orquestra-sdk`` and all of its dependencies.
 Note that ``orquestra-sdk`` also includes the :doc:`Orquestra command-line interface <../quickref/cli-reference>`.
 
+Note:
+
+.. code-block:: rst
+
+    When running the command above, you may observe an issue with it trying to install both 0.64.0 and 0.63.0
```
Comment (SebastianMorawiec) [inline projects/orquestra-sdk/docs/tutorials/installing-macos-linux.rst:25]: Suggestion: This line of document will become very outdated rather quickly (even now we have 0.65 already).

If I might suggest: I wouldn't focus on specific versions, I'd rather see a helpful non-version specific tips like:
"dependency conflicts error of 'ray==X.Y.Z; extra == "all" might suggest problems with Python versions. Right Now its best to have your environment at Python version 3.11.X (you can check your Python version using `python --version`

Also great comment about virtual environment! Thats very useful as well

**Comment by SebastianMorawiec** [comment]
Nitpick: If you could also update Changelog file - its really useful for us when publishing user-facing changelogs.
Nitpick: If you could also take a look at the checklist in the PR description and mark all [X] if everything is fulfilled :)

**Review by SebastianMorawiec — CHANGES_REQUESTED** [review]
Added few suggestions/nitpicks

**Comment by petermin123** [comment]
Thank you for the suggestions Sebastian. Working on them right now!

**Comment by petermin123** [comment]
Addressed the suggestions and nitpicks. Added the change under `Unreleased/Docs` of the changelog.

**Review by SebastianMorawiec — APPROVED** [review]
LGTM

**Pull request merged**